builtin/credential/cert/backend.go

```diff
@@ -22,12 +22,16 @@ func Backend() *backend {
 		Help: backendHelp,
 
 		PathsSpecial: &logical.Paths{
+			Root: []string{
```

Comment:
Don't make this `Root` -- we're removing `Root` paths nearly everywhere in favor of the normal ACL system.